frontend/src/pages/MapView.tsx

```diff
@@ -23,6 +26,32 @@ const MapView = () => {
   const [markers, setMarkers] = useState<marker[]>([]);
   const [showMarkers, setShowMarkers] = useState<boolean>(false);
   const [userMarker, setUserMarker] = useState<marker>();
+  const [offCanvas, setOffCanvas] = useState<boolean>(false);
+
+  const [barInfo, setBarInfo] = useState<barMenuInfo>({
+    name: "NULL",
+    display_phone: "NULL",
+    rating: "0.0",
+    location: {
+      address1: "NULL",
+    },
+    image_url:
+      "https://cdn1.vectorstock.com/i/1000x1000/31/20/image-error-icon-editable-outline-vector-30393120.jpg",
```

Comment:
Please download the image and use it as the stock image. If the user has no internet then the image wont load.